### PR TITLE
feat(plugins): add new EE plugins

### PR DESCRIPTION
**Pull request by travispamaral**
Style EE cards with blue background and ribbon. Add 3 new plugins as well as set existing EE
plugins as EE

### CODE REVIEW

**Comment by coopr** [comment]
The Edge Caching icon could really use reworking - it currently implies that one thing gets through, two others are blocked. A better thing for it to imply is that there are three requests, two of which are served by Kong's Cache, and one of which goes to the upstream for processing. I don't know how you draw that :) but I know that the current drawing looks too much like things are being blocked.

File: app/_data/plugins.yml [hunk 1]
```diff
@@ -49,6 +51,14 @@
 - name: Traffic Control
   info: Manage, throttle and restrict inbound and outbound API traffic
   plugins:
+    - name: Edge Caching
+      info: Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```
Comment (coopr) [inline app/_data/plugins.yml]: info: Cache and serve commonly-requested responses in Kong.

File: app/_data/plugins.yml [hunk 2]
```diff
@@ -49,6 +51,14 @@
...
+      enterprise: true
+
+    - name: Rate Limiting
+      info: Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```
Comment (coopr) [inline app/_data/plugins.yml]: info: Upgrades Kong Community Edition rate limiting with more flexibility and higher performance.

File: app/_data/plugins.yml [hunk 3]
```diff
@@ -86,6 +96,10 @@
   info: Transform request and responses on the fly on Kong
   plugins:
     - name: Request Transformer
+      info: Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```
Comment (coopr) [inline app/_data/plugins.yml]: info: Use powerful regular expressions, variables and templates to transform API requests.

File: app/plugins/ee-edge-caching.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+header_icon: /assets/images/icons/plugins/rate-limiting.png
+breadcrumbs:
+  Plugins: /plugins
+nav:
+  - label: Getting Started
+    items:
+      - label: Requesting Access
+---
+
+Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minus aliquid ea assumenda ipsa iure officia obcaecati quod, cupiditate exercitationem mollitia animi veritatis eum deleniti, et libero quos cumque consequuntur dolore.
```
Comment (coopr) [inline app/plugins/ee-edge-caching.md]: The Edge Caching plugin for Kong Enterprise Edition makes it fast and easy to configure caching of responses and serving of those cached responses to matching requests.

File: app/plugins/ee-edge-caching.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+</div>
+
+----
+
+## Requesting Access
+
+This plugin is only available with a [Mashape Enterprise](/enterprise)
+subscription.
+
+If you already are a Mashape Enterprise customer, you can request access to the
```
Comment (coopr) [inline app/plugins/ee-edge-caching.md]: Remove lines 30 and 31 - and the similar lines from all the other EE plugins.

File: app/plugins/ee-rate-limiting.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+header_icon: /assets/images/icons/plugins/rate-limiting.png
+breadcrumbs:
+  Plugins: /plugins
+nav:
+  - label: Getting Started
+    items:
+      - label: Requesting Access
+---
+
+Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minus aliquid ea assumenda ipsa iure officia obcaecati quod, cupiditate exercitationem mollitia animi veritatis eum deleniti, et libero quos cumque consequuntur dolore.
```
Comment (coopr) [inline app/plugins/ee-rate-limiting.md]: The Rate Limiting plugin for Kong Enterprise Edition is a re-engineered version of the incredibly popular Community Edition Rate Limiting plugin, with greatly enhanced configuration options and performance.

File: app/plugins/ee-request-transformer.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+header_icon: /assets/images/icons/plugins/EE-request-transformer.png
+breadcrumbs:
+  Plugins: /plugins
+nav:
+  - label: Getting Started
+    items:
+      - label: Requesting Access
+---
+
+Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minus aliquid ea assumenda ipsa iure officia obcaecati quod, cupiditate exercitationem mollitia animi veritatis eum deleniti, et libero quos cumque consequuntur dolore.
```
Comment (coopr) [inline app/plugins/ee-request-transformer.md]: The Request Transformer plugin for Kong Enterprise Edition builds on the Community Edition version of this plugin with enhanced capabilities to match portions of incoming requests using regular expressions, save those matched strings into variables, and substitute those strings into transformed requests via flexible templates.

**Comment by travispamaral** [comment]
@coopr I've updated with your text as well as replaced the Customer.io newsletter form with Pardot

**Pull request merged**